packages/forge/blocks/segment/actions/page.ts

```diff
@@ -0,0 +1,78 @@
+import { createAction, option } from '@typebot.io/forge'
+import { Analytics } from '@segment/analytics-node'
+import { auth } from '../auth'
+
+export const page = createAction({
+  auth,
+  name: 'Track Chatbot View',
```

Comment:
Can you make this action more generic? It should interface the `analytics.page` function. So maybe a `Track Page` action or else

Reply:
Depends on whether we want this to be clear to Segment users that this maps to the 'page' API event in Segments API. I originally named it trackPage but renamed it to page because that's what it is in Segments API

Reply:
changed it to trackPage